src/Microsoft.DotNet.Build.CloudTestTasks/FilterForUpload.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    public sealed class FilterForUpload : Task
+    {
+        [Required]
+        public string TestOutputDirListsLocation { get; set; }
+
+
+        [Required]
+        public string FilterToTestTFM { get; set; }
```

Comment:
I would love to see a big comment explaining why we're bringing in yet another variable with TestTFM and OSGroup to the tasks here.  It might convince me that this is actually necessary.